src/compiler/checker.ts

```diff
@@ -13794,15 +13794,19 @@ namespace ts {
         }
 
         function checkClassForDuplicateDeclarations(node: ClassLikeDeclaration) {
-            const getter = 1, setter = 2, property = getter | setter;
+            const enum Meaning {

```

Comment:
is `Accessor` already taken? (this is low priority, if you want to check in as-is, it's fine)